PathingAPI/data_config.json

```diff
@@ -0,0 +1 @@
+{"Version":7,"Root":"../json/"}
```

Comment:
can you put the `data_config.json` file into the `.gitignore` list and remove this. please

Reply:
done